### DISCUSSION TITLE
Add buttons for large chests

### DISCUSSION THREAD

**Discussion by OmerBenGera** [BG-Software-LLC/WildChests#41] [Pending Feature Requests]
Add buttons for previous and next pages for large chests

**Comment by Skizzles** [comment]
Reason for adding:

Bedrock users can not click outside of the GUI to switch pages
This would allow easier understanding of the Large Chest
Bonus:

You already have some pretty good menu systems from SSB so probably would be semi decent to implement.

**Reply by ELIAS7281** [reply]
Yes this is much needed, it should be added for all chests that allows extra pages in the config
